src/mobile/src/ui/components/SettingsContent.js

```diff
@@ -134,7 +135,7 @@ export const renderSettingsRows = (rows, theme) => {
                                     <View style={styles.content}>
                                         <Text style={[styles.titleText, textColor]}>{row.name}</Text>
                                         {row.currentSetting && (
-                                            <Text numberOfLines={1} style={[styles.settingText, textColor]}>
+                                            <Text numberOfLines={1} style={[styles.settingText, textColor, marginLeft]}>
```

Comment:
The `marginLeft` can just be included here: https://github.com/iotaledger/trinity-wallet/pull/972/files#diff-2e87c3da0f3ff562affa311e504e0161R102

```
settingText: {
        fontFamily: 'SourceSansPro-Light',
        fontSize: Styling.fontSize3,
        backgroundColor: 'transparent',
        marginLeft: 5,
    },
```

Reply:
If you want to completely fix the issue, I found that this helps:

```
settingText: {
        fontFamily: 'SourceSansPro-Light',
        fontSize: Styling.fontSize3,
        backgroundColor: 'transparent',
        marginLeft: 5,
        flex: 1,
        textAlign: right,
   },
```
So it will look like this:
![image](https://user-images.githubusercontent.com/19519564/51511088-e476c180-1dcd-11e9-8613-5991a7345b09.png)